hphp/runtime/vm/jit/vasm-simplify.cpp

```diff
@@ -196,6 +196,37 @@ bool simplify(Env& env, const setcc& vsetcc, Vlabel b, size_t i) {
   });
 }
 
+template <typename Reg>
+bool operand_one(Env& env, Reg op) {
```

Comment:
@mxw - Ok, got it.  I thought you were suggesting something else.  Having a separate file will make the boundaries cleaner.